### ISSUE TITLE
ADX Momentum Indicator logic appears to be backwards.

### ISSUE THREAD

**Issue by CodaCarlson** [freqtrade/freqtrade-strategies#94]
After looking at the ADXMomentum indicator, the logic appears to be slightly backwards from the quoted source. Below is what i believe it is supposed to look like but please double check.

```
    def populate_buy_trend(self, dataframe: DataFrame, metadata: dict) -> DataFrame:
        dataframe.loc[
            (
                    (dataframe['adx'] > 25) &
                    (dataframe['mom'] > 0) &
                    (dataframe['plus_di'] > 25) &
                    (dataframe['plus_di'] > dataframe['minus_di'])

            ),
            'buy'] = 1
        return dataframe

    def populate_sell_trend(self, dataframe: DataFrame, metadata: dict) -> DataFrame:
        dataframe.loc[
            (
                    (dataframe['adx'] > 25) &
                    (dataframe['mom'] < 0) &
                    (dataframe['minus_di'] > 25) &
                    (dataframe['plus_di'] < dataframe['minus_di'])

            ),
            'sell'] = 1
        return dataframe
```

Thanks for all the strategies. Keep up the good work.

**Comment by xmatthias** [comment]
Yes, that seems correct to me.

I've fixed this now. Thanks for noticing!

**Comment by PythonCommander** [comment]
yes, I noticed that as well. i was quite confused but thought to myself that maybe the creator of the strategy knew something I didn't know. So i created a hyperopt file of the exact strategy to optimize it as intended. When I ran it through the hyperopt the computer found amazing profits with it. I am on day 4 of the optimized strategy trading live, and it has 73/0 win loss rate. It seems to be catching huge pumps, and has tripled in profits of my stake amount.
